relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -63,6 +63,8 @@ pub enum SpanTagKey {
     FileExtension,
     /// Span started on main thread.
     MainThread,
+    /// The SDK integration (e.g., openai) that created the span. Not yet well supported by SDKs.
+    SdkIntegration,
```

Comment:
What does "Not yet well supported by SDKs" mean? The field is not documented in [dev docs](https://develop.sentry.dev/sdk/event-payloads/span/), could you link to the document where the protocol and use cases are defined?

I wonder if this is limited to a single integration (and can be typed as a string) or multiple (and typed as a list of strings), what the tag value should be in that last case, and whether this is the best name for it.

Reply:
I couldn't find any existing fields (we were considering `domain`) that matched this behavior

Reply:
https://github.com/getsentry/develop/pull/1226